data/gegno/gegno outfits.txt

```diff
@@ -719,6 +719,49 @@ effect "acuit hit"
 	"velocity scale" -0.004
 
 
+outfit "Acumen Bolt Cannon"
+	plural "Acuit Artillery"
```

Comment:
Shouldn't this be `plural "(Acumen) Bolt Cannons"`?

Reply:
Yes. This was copied from the Acuit before I changed stats. Thanks for pointing it out.